build/cmake/package_tester/modules/test_args.sh

```diff
@@ -0,0 +1,47 @@
+#!env bash
+
+if [ -z ${test_args_sh_included+x} ]
+then
+    test_args_sh_included=1
+
+    source ${source_dir}/modules/util.sh
+
+    test_args_usage() {
+        cat <<EOF
+usage: pkg_test.sh [-h] files...
```

Comment:
I can't seem to find pkg_test.sh

Reply:
the `pkg` part is meant to refer to either `deb` or `rpm`. But I changed this to make it a bit more generic (and more clear). Although this script is not meant to be called by a human.